google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettingsTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.stub;
+
+import com.google.api.gax.core.CredentialsProvider;
+import com.google.api.gax.grpc.InstantiatingGrpcChannelProvider;
+import com.google.bigtable.admin.v2.InstanceName;
+import com.google.cloud.bigtable.data.v2.BigtableDataSettings;
+import com.google.common.truth.Truth;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+
+@RunWith(JUnit4.class)
+public class EnhancedBigtableStubSettingsTest {
+  @Test
+  public void instanceNameIsRequiredTest() {
+    EnhancedBigtableStubSettings.Builder builder = EnhancedBigtableStubSettings.newBuilder();
+
+    Throwable error = null;
+    try {
+      builder.build();
+    } catch (Throwable t) {
+      error = t;
+    }
+
+    Truth.assertThat(error).isInstanceOf(IllegalStateException.class);
```

Comment:
I think the recommendation is to `import static com.google.common.truth.Truth.assertThat` and just use `assertThat` in the tests.